modules/calendar/config/locales/js-en.yml

```
@@ -6,3 +6,4 @@ en:
       title: 'Calendar'
       too_many: 'There are %{count} work packages in total, but only %{max} can be shown.'
       unsaved_title: 'Unnamed calendar'
+      calendars_label: 'Calendars'
```

Usually we follow the syntax like this _lable_name_plural_
```suggestion
      label_calendar_plural: 'Calendars'
```